SystemServices/SystemServices.cpp

```diff
@@ -317,117 +316,114 @@ namespace WPEFramework {
              * @brief Invoking Plugin API register to WPEFRAMEWORK.
              */
 #ifdef DEBUG
-            registerMethod("sampleSystemServiceAPI", &SystemServices::sampleAPI, this);
+            RegisterMethod(this, {1,2}, "sampleSystemServiceAPI", &SystemServices::sampleAPI, this);
 #endif /* DEBUG */
-            registerMethod("getDeviceInfo", &SystemServices::getDeviceInfo, this);
+            RegisterMethod(this, {1,2}, "getDeviceInfo", &SystemServices::getDeviceInfo, this);
```

Comment:
We don't need these changes. the default registerMethod() takes care of making this API available on all available versions. So only the APIs that are registered on versions 2 or above using GetHandler() need to be made available on version 1. So basically replace GetHandler(2)->registerMethod() with registerMethod().